### PR TITLE
Optionally bundle js into multiple bundles

### PR DESCRIPTION
**Pull request by thomasdegry**
Just the start, didn't test this yet. But as I implemented this on grandstand wanted to add it into mo-static as well. Will follow up on this.

### CODE REVIEW

**Comment by thomasdegry** [comment]
One thing I can confirm is: watchify is broken with this implementation. Seems like there is a way around it using: https://github.com/reducejs/reduce-js

**Comment by thomasdegry** [comment]
@akrawchyk fixed the grandstand reference!

File: {{ cookiecutter.repo_name }}/gulp/build.js [hunk 1]
```diff
@@ -12,30 +12,40 @@ import buffer from 'vinyl-buffer'
...
-  .transform('eslintify', { continuous: true })
-  .transform('babelify')
-  .transform(envify(config.get()))
-  .transform('uglifyify')
+let bundler = browserify({
+  entries: [
+    'app.js',
+  ],
+  fullPaths: true,
+  basedir: './{{ bookiecutter.repo_name }}/static/js/',
```
Comment (akrawchyk) [inline {{ cookiecutter.repo_name }}/gulp/build.js]: bookiecutter -> cookiecutter

**Comment by akrawchyk** [comment]
Closing in favor of #71

**Pull request closed without merging**